packages/gatsby-transformer-remark/src/extend-node-type.js

```diff
@@ -105,7 +105,7 @@ module.exports = (
         return await ASTPromiseMap.get(cacheKey)
       } else {
         const ASTGenerationPromise = new Promise(async resolve => {
-          if (process.env.NODE_ENV === `production` && !fileNodes) {
+          if (process.env.NODE_ENV === `development` || !fileNodes) {
```

Comment:
Could You use ``process.env.NODE_ENV !== `production` ``? This is what we use in our codebase, and it's easier to find `gatsby build / develop` differences if detection method is consistent

Reply:
Okay, that makes sense, will do.

Reply:
Since we're negating both would it be preferable to be something like this?
`if (!(process.env.NODE_ENV === `production && fileNodes)) {`